curvefs/src/mds/topology/topology_manager.cpp

```diff
@@ -125,7 +125,12 @@ void TopologyManager::RegistMetaServer(const MetaServerRegistRequest *request,
         }
     }
 
-    MetaServer metaserver(metaServerId, request->hostname(), token, serverId,
+    std::string hostname;
+    if (request->has_hostname()) {
```

Comment:
not need, request->hostname() is empty string if hostname not set.

Reply:
ok